indexer-app/src/cmr/indexer/data/index_set.clj

```diff
@@ -421,6 +430,14 @@
           :usage-relevancy-score m/int-field-mapping}
          spatial-coverage-fields))
 
+(defmapping deleted-granule-mapping :deleted-granule
+  "Defines the elasticsearch mapping for storing granules"
+  {:concept-id (-> m/string-field-mapping m/stored m/doc-values)
+   :delete-time (-> m/string-field-mapping m/stored m/doc-values)
```

Comment:
I think we eventually want to search deleted granules by delete time. We should index the `delete-time` as a date field for that reason.